core/string/ustring.cpp

```diff
@@ -1022,6 +1022,13 @@ static _FORCE_INLINE_ signed char naturalnocasecmp_to_base(const char32_t *p_thi
 		while (*p_this_str) {
 			if (!*p_that_str) {
 				return 1;
+			} else if (*p_this_str == '.' || *p_that_str == '.') {
+				if (*p_this_str++ != '.') {
```

Comment:
Such usage of `++` doesn't look very readable.

Reply:
It would require either additional increments in both paths or storing a temporary, either way it's a bit low readability